src/main/java/cafeboard/Board/Board.java

```diff
@@ -0,0 +1,50 @@
+package cafeboard.Board;
+
+import cafeboard.Post.Post;
+import jakarta.persistence.*;
+
+
+import java.util.List;
+
+@Entity
+public class Board {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long boardId;
+
+    private String title;
+
+    @OneToMany(mappedBy = "board", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Post> post;
+
+
+    public Board() {
+    }
```

Comment:
사소한 거긴 한데, 개발자가 직접 사용할 생성자가 아니기 때문에 public보다 한 단계 낮춰서 protected로 두면 더 좋습니다.
```suggestion
    protected Board() {
    }
```